website/docs/admin_import_openpype_project.md

```diff
@@ -0,0 +1,217 @@
+dbonly = True
+pack_project(project, destination_dir, dbonly)
+```
+
+![pack op project from terminal](assets/import_openpype/pack_op_project_from_console.png)
+
+### II. Setting up ASH
+
+First requirement is to have [ASH](https://github.com/ynput/ash) set up and running. ASH stands for *AYON Service Host* and it takes
+care for handling AYON Services. It Periodically checks the services declared in th AYON and starts
+them if they are not running. Since OpenPype Importer Service can be started with ASH, let's firstly set it up:
+
+```sh
+git clone https://github.com/ynput/ash.git
+cd ./ash
+# to build the Docker image
+docker build -t ynput/ayon-ash:latest
+```
+
+If you want ASH to be part of your AYON Docker stack, make sure that `docker-compose.yml` in your
+[ayon-docker](https://github.com/ynput/ayon-docker.git) has something like this in:
+
+```yaml
+worker:
+    image: ynput/ayon-ash:latest
+    hostname: worker01
+    restart: unless-stopped
+    depends_on:
+      - server
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock"
+    environment:
+      - "AYON_API_KEY=xxxxxxxxxxxxxxxxxxxxxxx"
+      - "AYON_SERVER_URL=http://172.18.0.4:5000"
+```
+
+*NOTE:* you can (and should) customize this to fit your needs. Be aware that `AYON_SERVER_URL` must
+be reachable from withing the Docker network, otherwise you'll get connection errors.
+
+With these changes, you can run it with:
+
+```sh
+docker compose up --detach --build worker
+```
+
+If you rather like to run it manually with `docker`, this command will do the same:
+
+```sh
+# to run the image
+docker run -rm -ti \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    --hostname worker01
+    --env AYON_API_KEY=xxxxxxxxxxxxxxxxxxxxxxx
+    --env AYON_SERVER_URL=http://172.18.0.4:5000
+    ynput/ayon-ash
+```
+
+For development, you can also run it directly from Python 3.10+ using Poetry:
+
+```sh
+poetry run -m ash
+```
+
+:::info
+**<font size="4">Alternatively,</font>** you can run ASH as a part of your main Ayon stack as mentioned in this community guide [AYON Beta Testing - Minimal step by step guide](https://community.ynput.io/t/ayon-beta-testing-minimal-step-by-step-guide/699#ayon-server-host-ash-18)
+
+This can be much easier to set up if you are not familiar with docker.
+:::
+
+## Project Migration Steps
+
+### I. Get OpenPype Importer Addon
+You two options, choose your preferred option.
+
+#### Option 1: Just Download It!
+Head to [`ayon-openpype-import` releases github page](https://github.com/ynput/ayon-openpype-import/releases) and download the latest release.
+
+#### Option 2: Export the addon yourself
+> This is useful when you want to test the current development code!
+
+First, clone repository addon:
+
+```sh
+git clone https://github.com/ynput/ayon-openpype-import
+```
+
+Create server addon from the sources by running `create_package.py` script:
+
+```sh
+python ./create_package.py
```

Comment:
This doesn't exist in the repo, but I assume it will be added soon.

Reply:
yeah, there is open PR just for that